src/test/resources/features/HomePage.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Home Page Verification
+
+  Scenario: Verify Help Center page title
+    Given User is on the Home Page
+    When User clicks on the Help Center link
+    Then User is navigated to the Help Center page
```

Comment:
Lets changes this step to more clear. For example: "Verify user is navigated to the Help Center page"